app/controllers/page_list_controller.rb

```diff
@@ -23,11 +16,11 @@ def update
         raise StandardError, "Save unsuccessful"
       end
     else
-      render :edit, status: :unprocessable_entity
+      render :index, status: :unprocessable_entity
```

Comment:
Just curious and surely some convention I'm unaware of, before it was referencing `:edit` template within the pages_list controller https://github.com/alphagov/forms-admin/blob/main/app/views/page_list/edit.html.erb. Now we seem to want it to render the `index` view of `pages` which doesn't exist https://github.com/alphagov/forms-admin/tree/main/app/views/pages. The Rails docs also seem to imply that if you want one controller to render the template from another controller you have to give it more information https://guides.rubyonrails.org/layouts_and_rendering.html#rendering-an-action-s-template-from-another-controller so I would expect something like `render "pages/index". 

Since it builds and CI passes I assume it does and it's using some convention that I don't get yet, but also no tests were updated in this commit so perhaps its not being tested? Anyway thought it worth asking :)

Reply:
Oh thats a great catch... 👏 

I was using RubyMines "Refactor" feature when you move/rename files it tries to be helpful which is great in this instance BUT we obviously dont seem to be testing the unhappy path which is why tests are all passing. 

See code coverage report as proof that we aren't testing unhappy path...
![image](https://user-images.githubusercontent.com/3441519/220060619-28d45b7d-c31d-481f-85a4-b899aa89cb9a.png)

Reply:
I'm confused generally about the difference between `pages_controller` and `page_list_controller`.

It looks like from the routes file in the main branch that the pages controller handles all changes that might be done via the page list form, including reordering and marking the questions as complete, which makes sense to me.

However in this PR you're moving most of the functionality from page list controller to pages, leaving just marking as complete ... I personally think it makes sense to either go the whole way and remove the page list controller (put the confirmation update method in the pages controller with the others?), or just leave it as it is and keep the split. Having the functionality for the pages list form split across two controllers seems a little strange to me.

Reply:
@danworth see 2ebfcb9c515afedff7355a65a026e334ab726698, basically, I moved some of the functionality out of the controller to simplify it. By abstracting this bit of code and getting the controller to only be responsible to managing which page the user sees next makes it easier to test. 

"Thin controllers are always better than fat controllers"


100% now for both controller specs and the model 
![image](https://user-images.githubusercontent.com/3441519/220476875-58751c36-bb19-480b-8be4-dc63582e36bb.png)

![image](https://user-images.githubusercontent.com/3441519/220476933-8ba6a5e8-2ad6-4ffb-8ae7-e97c2e74c514.png)

Reply:
@lfdebrux 100% agree, this is something we were debuting about when we make the changes to move update action. I've now completely got rid of `page_list_controller` which didn't make sense as we dont have a pages_list resource. done as part of 8c301bf705288d547fd485f9fbe88d793fee0932